src/main/scala/nl.knaw.dans.easy.deposit/logging/package.scala

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.deposit
+
+import javax.servlet.http.{ HttpServletRequest, HttpServletResponse }
+import org.scalatra.ActionResult
+
+//TODO: candidate for dans-scala-lib
+
+/**
+ * Provides standard logging for servlet requests and responses masking sensitive data.
+ *
+ * ==Requests==
+ *
+ * All requests are logged at info level by adding a trait to servlets:
+ * {{{
+ *   class MyServlet extends ScalatraServlet with RequestLogger {???}
+ * }}}
+ * To change the log level (or use another logger) define your own RequestLogger like:
+ * {{{
+ *   trait DebugRequestLogger extends RequestLogFormatter {
+ *     this: ScalatraBase =>before() { logger.debug(formatRequestLog) }
```

Comment:
With `logger.debug(request.getXxx)` you not only override the logger but the formatter on the flight too. Same for the responses.

I don't like that. But we are already way off-topic. For now we have to live with it but it means the package is not library-ready.

Reply:
Not sure what the problem about `logger.debug` is. But let's take this to another time and get this PR finished ASAP.

Reply:
The problem is that besides overriding _how_ we log, we can at the same time override _what_ we log. When doing the latter, we completely ignore the formatter extended by the logger. That feels like a wrong inheritance.